### DISCUSSION TITLE
Fixing issue by adding line to different file

### DISCUSSION THREAD

**Discussion by RQDineshkumar** [eslint/eslint#15347] [Q&A] [answered]
I have wrote a plugin with a rule that checks whether they key used for i18n in javascript files exists in locale json files. Now I want to implement auto fix for that rule. The `fixer` object doesn't have any methods to insert text into different file than current one. Is the only way is to manually write into JSON file...?

**Comment by nzakas** [comment, marked as answer]
Fixers are only meant to work on the current file. If you want to do that, you’ll need to do it manually.

**Reply by RQDineshkumar** [reply]
Figured 🤧
